components/team_sidebar/team_sidebar_controller.jsx

```diff
@@ -135,7 +135,10 @@ export default class TeamSidebar extends React.PureComponent {
         );
 
         return (
-            <div className={classNames('team-sidebar', {'move--right': this.props.isOpen})}>
+            <div
+                className={classNames('team-sidebar', {'move--right': this.props.isOpen})}
+                role='navigation'
```

Comment:
With axe, I've got error of "Ensures landmarks are unique". Suggested fix is "The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable"

<img width="783" alt="Screen Shot 2020-01-07 at 4 34 04 PM" src="https://user-images.githubusercontent.com/5334504/71880606-af44ba00-316b-11ea-9cf3-20796f06fe33.png">